src/user/user.schema.js

```diff
@@ -0,0 +1,24 @@
+import { schema } from 'normalizr';
+import omit from 'lodash/omit';
+
+export const userSchema = new schema.Entity(
+  'users',
+  {},
+  {
+    idAttribute: user => user.login.toLowerCase(),
+    processStrategy: entity =>
+      omit(entity, [
```

Comment:
All this fields can be guessed by having just the username. We remove them from the object to minimize our footprint